packages/components/src/template-registry.ts

```diff
@@ -289,6 +292,16 @@ export default interface HdsComponentsRegistry {
   'Hds::Stepper::Task::Indicator': typeof HdsStepperTaskIndicatorComponent;
   'hds/stepper/task/indicator': typeof HdsStepperTaskIndicatorComponent;
 
+  // Tabs
+  'Hds::Tabs': typeof HdsTabsComponent;
+  'hds/tabs': typeof HdsTabsComponent;
+
+  'Hds::Tabs::Panel': typeof HdsTabsPanelComponent;
+  'hds/tabs/panel': typeof HdsTabsPanelComponent;
+
+  'Hds::Tabs::Tab': typeof HdsTabsTabComponent;
+  'hds/tabs/tab': typeof HdsTabsTabComponent;
```

Comment:
```suggestion
  'Hds::Tabs::Tab': typeof HdsTabsTabComponent;
  'hds/tabs/tab': typeof HdsTabsTabComponent;

  'Hds::Tabs::Panel': typeof HdsTabsPanelComponent;
  'hds/tabs/panel': typeof HdsTabsPanelComponent;
```

Reply:
I'll keep these in alphabetical order unless you want to change it up still?